src/main/java/eu/enmeshed/EnmeshedMessagingService.java

```diff
@@ -82,7 +82,6 @@ public String sendAuthenticationRequest(
                     AuthenticationRequestItem.builder()
                         .title(displayTitle)
                         .description(displayText)
-                        .mustBeAccepted(mandatory)
```

Comment:
the comment is gone, but this is still not re-added.

Reply:
Added